teachertool/src/transforms/setEvalResultOutcome.ts

```diff
@@ -0,0 +1,13 @@
+import { stateAndDispatch } from "../state";
+import * as Actions from "../state/actions";
+import { CriteriaEvaluationResult } from "../types/criteria";
+
+export function setEvalResultOutcome(criteriaId: string, result: CriteriaEvaluationResult) {
```

Comment:
I think we should rename the `CriteriaEvaluationResult` enum to something like `EvaluationStatus`. I keep getting confused because I think `CriteriaEvaluationResult` is the type for the status + notes object, but then it's actually just the status. And in some places (like `newCriteriaEvalResult` below), we kind of are using the term `CriteriaEvalResult` to refer to the whole object. So just to minimize confusion, I think it'd help to rename the enum.